pubspec.yaml

```diff
@@ -8,4 +8,7 @@ author: Devon Carew <devoncarew@google.com>
 environment:
   sdk: '>=1.9.0 <2.0.0'
 dependencies:
+  args: '^0.13.1'
   logging: '^0.11.0'
+  shelf: '^0.6.1+2'
```

Comment:
It's unfortunate that we can't separate out the `lib/` dependencies from the `bin/` ones.